### PR TITLE
Rebased pr73824

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Updates from `ansible-base` to `ansible-core` where appropriate. 

Thanks to @KreolUrs for the original work! 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION
The original PR included some changes that I rolled back, because `ansible-base` did exist in version 2.10. We cannot undo that, so we must use `ansible-base` in some places, especially in the porting guides.

### CODE REVIEW

**Comment by acozine** [comment]
Based on #73824.

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 1]
```diff
@@ -530,7 +530,7 @@ Packaging Ansible or wanting to build a local package yourself, but don't want t
 
 .. note::
 
-	If you are creating your own Ansible package, you must also download or package ``ansible-base`` as part of your Ansible package. You can download it as https://pypi.python.org/packages/source/a/ansible-base/ansible-base-{{VERSION}}.tar.gz.
+	If you are creating your own Ansible package, you must also download or package ``ansible-core`` as part of your Ansible package. You can download it as https://pypi.python.org/packages/source/a/ansible-core/ansible-core-{{VERSION}}.tar.gz.
```
Comment (samccann) [inline docs/docsite/rst/installation_guide/intro_installation.rst:534]: I can't get this url to work in any variation.

Comment (sivel) [reply docs/docsite/rst/installation_guide/intro_installation.rst:534]: ansible-core is only for 2.11+

We have a beta2 up that seems to work:

https://pypi.python.org/packages/source/a/ansible-core/ansible-core-2.11.0b2.tar.gz

Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst:534]: Thanks @samcann @sivel. I'll try adding a list here to show the different versions and the URLs that apply to them.

Comment (lmalivert) [reply docs/docsite/rst/installation_guide/intro_installation.rst:534]: @acozine Reviewed the PR and looks like you caught the mistakes. LGTM.

Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst:534]: Addressed in 578a6191ef0bb7a2fa31e0f2dc5e40065afc67ed.

**Review by samccann — APPROVED** [review]

**Pull request merged**